cms/envs/common.py

```diff
@@ -793,6 +793,7 @@
 
 OPTIONAL_APPS = (
     'mentoring',
+    'problem_builder',
```

Comment:
OK, I added it to the lms/envs/common.py. I just amended the commit with that one-line change.